tests/integration/test.prefix.js

```diff
@@ -38,7 +38,9 @@ describe('test.prefix.js', function () {
 if (typeof process !== 'undefined' &&
     !process.env.LEVEL_ADAPTER &&
     !process.env.LEVEL_PREFIX &&
-    !process.env.ADAPTER) {
+    !process.env.ADAPTER &&
+    // fails on windows with EBUSY - "resource busy or locked", not worth fixing
```

Comment:
Ah yeah I was running WebStorm at the same time. That may be the issue.

In any case, this test is reeeeeaaally not important enough to justify a lot of workarounds to get it passing.